Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.yml

```diff
@@ -29,15 +36,16 @@ configuration:
   name: proxy
   required: false
   type: 8
-description: A threat intelligence and investigation platform for domain names, IP addresses, email addresses, name servers and so on.
+description: A threat intelligence and investigation platform for domain names, IP
+  addresses, email addresses, name servers and so on.
 display: DomainTools Iris
 name: DomainTools Iris
 script:
   commands:
   - arguments:
     - default: true
       description: The domain name.
-      isArray: false
+      isArray: true
```

Comment:
the api supports that?

Reply:
I think the api supports but the integration doesn't. This change got in via the demisto-sdk `format` command. Will open a dev-bug on this.

Reply:
https://github.com/demisto/etc/issues/30208